core/chains/evm/client/errors.go

```diff
@@ -128,10 +128,9 @@ var arbitrum = ClientErrors{
 	L2Full:                regexp.MustCompile(`(: |^)(queue full|sequencer pending tx pool full, please try again)(:|$)`),
 }
 
-var optimism = ClientErrors{
-	L2FeeTooLow:  regexp.MustCompile(`(: |^)fee too low: \d+, use at least tx.gasLimit = \d+ and tx.gasPrice = \d+$`),
-	L2FeeTooHigh: regexp.MustCompile(`(: |^)fee too high: \d+, use less than \d+ \* [0-9\.]+$`),
-}
+// Optimism Bedrock introduced the same errors as geth
+// https://github.com/ethereum-optimism/op-geth/blob/optimism/core/error.go
+var optimism = geth
 
```

Comment:
this is gonna need to stay and be marked for removal later to support both versions at once